### PR TITLE
fix panic when user not found

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: controller/users.go [hunk 1]
```diff
@@ -321,7 +321,7 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
-			"user_name": identity.ID,
+			"user_name": id.String(),
```
Comment (sbose78) [inline controller/users.go]: let's call it `identity_id`

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-111-1`

File: controller/users.go (hunk 1 above)
Comment (alexeykazakov) [reply controller/users.go]: Makes sense :)
Done.

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-111-2`

**Pull request merged**